EazyE2E/Element/EzRootElement.cs

```diff
@@ -12,22 +12,22 @@ namespace EazyE2E.Element
 {
     public class EzRoot
     {
-        public EzElement RootElement;
+        private readonly EzProcess _process;
+        private EzElement _rootElement;
 
-        public EzRoot(string processName)
-        {
-            RootElement = Set(processName);
-        }
+        public EzElement RootElement => _rootElement ?? (_rootElement = GetRootElement());
 
         public EzRoot(EzProcess process)
         {
-            RootElement = Set(process.ProcessName);
+            _process = process;
```

Comment:
In the GetRootElement method, we get the root element of the specific process by the process's id.  This will ensure we are using the same process throughout.